app/code/Magento/Customer/Block/Adminhtml/Group/Edit.php

```diff
@@ -1,103 +0,0 @@
-<?php
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-namespace Magento\Customer\Block\Adminhtml\Group;
-
-use Magento\Customer\Api\GroupManagementInterface;
-use Magento\Customer\Api\GroupRepositoryInterface;
-use Magento\Customer\Controller\RegistryConstants;
-
-/**
- * Customer group edit block
- */
-class Edit extends \Magento\Backend\Block\Widget\Form\Container
```

Comment:
Please keep this block and mark as deprecated

Reply:
done